UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -581,15 +572,15 @@ public class Background : UndertaleObject, INotifyPropertyChanged, IDisposable
         /// <remarks>
         /// This attribute is UMT-only and does not exist in GameMaker.
         /// </remarks>
-        public bool TiledHorizontally { get => TileX > 0; set { TileX = value ? 1 : 0; OnPropertyChanged(); } }
+        public bool TiledHorizontally { get => tileX > 0; set { tileX = value ? 1 : 0; OnPropertyChanged(); } }
```

Comment:
Since we now the behaviour a little better, this should be also adjusted to be
`get => tileX != 0`